CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [v0.3.6] - 2024-02-05
+
```

Comment:
Doesn't the release PR itself (#808) need to go in here as well?

Reply:
We sometimes did and sometimes don't. It does not add a lot of value to include it, but often I added it for completenes. Will add it here as well.

Reply:
Decided to not add it, to reduce noise.